src/Microsoft.Identity.Client/Internal/Instance/Authority.cs

```diff
@@ -244,7 +244,7 @@ public static string CanonicalizeUri(string uri)
                 uri = uri + "/";
             }
 
-            return uri.ToLower();
+            return uri.ToLowerInvariant();
```

Comment:
I assume we also want an invariant version here. URIs cannot have ANSI characters.